demos/http_client_demo.cc

```diff
@@ -47,6 +48,7 @@ struct printer {
 int main(int ac, char** av) {
     app_template app;
     app.add_options()
+            ("https", bpo::value<bool>()->default_value(false), "Use HTTPS on port 443 (if off -- use HTTP on port 80)")
```

Comment:
nit, could use `bpo::bool_switch`, like:

```suggestion
            ("https", bpo::bool_switch(), "Use HTTPS on port 443 (if off -- use HTTP on port 80)")
```

Reply:
Even better, thanks!